plugins/modules/domain_record_info.py

```diff
@@ -1,193 +1,105 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
 
-"""This module contains all of the functionality for Linode Domain records."""
+"""This module allows users to retrieve information about a Linode VPC Subnet."""
```

Comment:
I think this should say `Linode Domain record` instead of `Linode VPC Subnet`.

Reply:
oops, nice catch!